### DISCUSSION TITLE
"This repo helped me with..."

### DISCUSSION THREAD

**Discussion by dpaulino** [jenius-apps/ambie#141] [General]
Hi friends.

One of my goals with Ambie was to create an open source UWP app that can serve as the ultimate "sample app" to help other developers. It's a real app with real customers, but it's simple enough for others to clone the repo and understand the architecture. But how do I know if I'm reaching this goal?

So this discussion is an experiment: If you have used Ambie's repo to learn something new about a topic (like MVVM, WinUI, etc) or maybe to copy some code, or maybe to use the code as inspiration for something else... If you used this repo in any way, I would love to hear from you. 

Simply reply to this discussion with something like "This repo helped me with... [insert your experience here]". This will help me understand if it's worth making more open source projects.

Thanks,
Daniel

**Comment by ahmed605** [comment]
This repo helped me understanding how some animations work, understanding some media management techniques, and understanding some APIs such as Uri/Protocol activation

I also plan to learn MVVM using this repo as example/sample for learning

**Comment by ismaelestalayo** [comment]
I recently started moving to MVVM and used your code alongisde the Community MVVM sample App to understand the best way to use it.

I also looked at the different folders you had and applied some of them to my project that I previously did not have like the `Constants`  folder with string keys I previously had hardcoded in multiple pages; or the `Services` folder which is my next step after I complete the MVVM refractor.

**Comment by extragloves** [comment]
I used this repo as WinUI inspiration for my first ever native Windows app. I think I found the app via Reddit as a recommendation for a beautiful WinUI 3 app. Now I'm using it for its intended purpose too :) Mad a PR to fix some broken Swedish as a thank you!
